buildtest/cli/__init__.py

```diff
@@ -82,6 +83,35 @@ def positive_number(value):
     return int_val
 
 
+def supported_color(value):
+    """Checks if input is positive number and returns value as an int type.
+
+    Args:
+        value (str): Specify an input number
+
+    Returns:
+        str: Return value as str type
+
```

Comment:
again same thing please update this docstring accordingly

Reply:
Updated input and return sections